server/src/main/java/org/elasticsearch/index/codec/PerFieldMapperCodec.java

```diff
@@ -44,6 +45,8 @@ public final class PerFieldMapperCodec extends Lucene99Codec {
     private final ES87BloomFilterPostingsFormat bloomFilterPostingsFormat;
     private final ES87TSDBDocValuesFormat tsdbDocValuesFormat;
 
+    private final ES812PostingsFormat ES812PostingsFormat;
```

Comment:
nit: use camel case for members?

```suggestion
    private final ES812PostingsFormat es812PostingsFormat;
```

Reply:
done